scripts/translation/libqa/SyncFileList.php

```diff
@@ -31,43 +31,40 @@ static function load()
         }
 
         $lang = trim( file_get_contents( $file ) );
-        $cache = __DIR__ . "/../../../temp/qaxml.files.$lang";
+        $cacheFilename = __DIR__ . "/../../../temp/qaxml.files.$lang";
 
-        if ( file_exists( $cache ) )
+        if ( file_exists( $cacheFilename ) )
         {
-            $data = file_get_contents( $cache );
+            $data = file_get_contents( $cacheFilename );
             return unserialize( gzdecode( $data ) );
```

Comment:
```suggestion
            return unserialize( gzdecode( file_get_contents( $cacheFilename ) ) );
```
Avoids ambiguous variable name